.github/workflows/default.yml

```diff
@@ -0,0 +1,154 @@
+---
+name: AnsibleCI
```

Comment:
Maybe the filename and name should be more descriptive of what the goal of this workflow is. "default" is not saying anything and "AnsibleCI" is not that helpful for a CI workflow in an Ansible role repo. Maybe "Full Deploy" or something like that?

Reply:
completely true.
I will change that.